### PR TITLE
Kernel errors experimental mode

### PR DESCRIPTION
**Pull request by terraqout**
Works well for me with iJulia, don't test well with other languages.
But maybe too verbose like with Julia0.5.

Give a try, please

### CODE REVIEW

File: lib/kernel.coffee [hunk 1]
```diff
@@ -55,15 +55,24 @@ class Kernel
...
+     stderr: (data, caption = "kernel answer") ->
+        detail = data.toString()
+        method = 'addInfo'
+        if /warning/gi.test detail
+            method = 'addWarning'
+        if /error/gi.test detail
+            method = 'addError'
+
+        console.error "kernel process received on stderr:", detail
+        atom.notifications[method] "Hydrogen: #{caption}",
```
Comment (willwhitney) [inline lib/kernel.coffee]: We should probably change this to `"<language> Kernel: #{caption}"`

**Comment by willwhitney** [comment]
If this is too verbose, we could throttle it and make any errors beyond the first in a window of time just say "More error messages received. Check the Developer Console for details." At least for now; at some point it would be cool to have a little panel where you can view all the messages from the kernel.

**Comment by terraqout** [comment]
@willwhitney fixed.

It isn't verbose if kernel is not. Well Julia is a chatterbox. In python or coffee I get only kernel errors.

**Comment by willwhitney** [comment]
We should deal gracefully with noisy kernels like Julia as well; I think the Notebook does something pretty concise in that case, and it'll be really annoying if lots of messages pop down in front of our code.

**Comment by terraqout** [comment]
Let's try. I as Julia user didn't see trouble in all this warning bloating. If someone find this annoying he can create a ticket and we'll discuss it again.

I thought error handling is kernel internal, so it should be managed there.

**Comment by willwhitney** [comment]
OK, sounds good. Merge away!

**Comment by terraqout** [comment]
Just leave this here: https://github.com/JuliaLang/julia/issues/4895

**Pull request merged**